### DISCUSSION TITLE
Focus open-source community efforts on a single package

### DISCUSSION THREAD

**Discussion by simon-liebehenschel** [aio-libs/aiocache#646] [General]
Hello @Dreamsorcerer @pshafer-als 

[aiocache](https://github.com/aio-libs/aiocache) was abandoned, but during the last years two other packages showed themselves as well-maintained and mature. I am talking about https://github.com/Krukov/cashews and https://pypi.org/project/HermesCache/

I have a proposal to pick an existing stable maintained caching package (from the two mentioned), and help to keep it stable. On my opinion, this is better than start to maintain again a legacy code while there is already a few packages with exactly the same functionality.

I do not know "aio-libs" team plans. Maybe you want to maintain a package which duplicates functionality of existing packages just to have full control on how logic will be implemented. 
Maybe "aio-libs" team may want to acquire "cashews" or "HermesCache" (of course, if that developers will agree).

My main point is that it is more efficient to work together on an open-source project rather everyone will develop the same functionality on its own.

What do you think?

**Comment by Dreamsorcerer** [comment]
I'm open to other collaborations, but this "legacy" code is pretty well written and has great test coverage, it didn't take much to get everything up to date again. The only things I'm looking to change are some small refactoring (listed under the 1.0 milestone) to make the API design a little cleaner.

**Comment by Dreamsorcerer** [comment]
Hermes says it has "experimental support for caching asyncio coroutines" and does not appear to be very active.

Both libraries, when glancing at the documentation, appear to have very similar designs to aiocache, including probably needing some of the same refactoring changes (like #609). So, yeah, might be nice to merge some efforts.

**Reply by simon-liebehenschel** [reply]
Sorry, but a few more questions. Is there at least one active (online, reachable) person with a "maintainer" permissions for this repository? If exactly you will stop working on this package for some reason, will it be abandoned again for years? Thank you.

**Reply by Dreamsorcerer** [reply]
I'm not sure it's visible if you're not a member, but there are 15 members in https://github.com/orgs/aio-libs/teams/aiohttp-dev who all have access.
